DepositeCalcTests/Pages/LoginPage.cs

```diff
@@ -34,5 +33,10 @@ public void Login(string login = "test", string password = "newyork1")
             PassworldFld.SendKeys(password);
             LoginBtn.Click();
         }
+
+        public bool IsOpened()
+        {
+            return LoginFld.Displayed;
```

Comment:
I did it, and everything worked as expected, and the test failed because the driver could not find the specified element on this (other) page. I don't understand what the problem is with this approach =( 
Previously, you recommended tracking changes in the URL or the presence (absence) of elements on the page when the page changes

Reply:
Would this be a better solution?
![image](https://user-images.githubusercontent.com/96728802/201054846-c50e183d-e438-49db-9dc9-4c0c5881beb3.png)

Reply:
If you are trying to find non-existed element with `FindElement` - you'll get NotFoundException. But, there is another solution: 
```
bool isElementPresent = FindElements(By...).Count > 0;
```
You can find all elements by locator and check that count of elements is greater than zero.

As for check by URL, it is also valid check.